qemu/tests/qemu_disk_img.py

```diff
@@ -96,9 +96,15 @@ def __md5sum(self, dst):
         return md5
 
     @error.context_aware
-    def save_file(self, dst):
+    def save_file(self, dst, guest_sync=False):
         error.context("save file('%s') md5sum in guest" % dst, logging.info)
         self.__create_file(dst)
+        if guest_sync:
+            login_timeout = int(self.params.get("login_timeout", 360))
+            session = self.vm.wait_for_login(timeout=login_timeout)
+            error.context("sync guest data after create it", logging.info)
```

Comment:
Better to write as error.context("sync guest data after copy it", logging.info).
The file is created on the host, then copied to guest.